main.tf

```diff
@@ -2,11 +2,16 @@ locals {
   datadog_integration_role_name = "DatadogAWSIntegrationRole"
   datadog_aws_account_id        = "464622532012"
 
-  install_log_forwarder = var.api_key != null && var.install_log_forwarder ? 1 : 0
+  install_log_forwarder  = var.api_key != null && var.install_log_forwarder ? 1 : 0
+  datadog_forwarder_yaml = data.http.datadog_forwarder_yaml_url.response_body
```

Comment:
If you can please move this to be with the other `datadog_` local variables (and sort it while you're at it) I think we're good! Nice improvement 👍

Reply:
Thanks, reorganized the local vars and sorted alphabetically.